### ISSUE TITLE
[Feature Request] Hide messages from blocked users in group chats. 

### ISSUE THREAD

**Issue by xWee** [23rd/tdesktop#59]
### Is your feature request related to a problem?

Telegram allows us to block users, but in group chats, blocked users' messages still annoys us. So there is some clients that can hide these messages, such as kotatogram fork in [AUR](https://aur.archlinux.org/packages/kotatogram-desktop-dynamic-bin/). I think Forkfram should add this feature, too.

### Describe the solution you'd like

.

### Describe alternatives you've considered

.

**Comment by 23rd** [comment]
To properly hide messages from a blocked user, we need to hide them in Groups, Discussions, Pinned messages, Shared Media, Media Viewer, and perhaps elsewhere.
I'm not sure this fits my project's "only some useful small features" concept.

Or do you just want to hide messages in conversations? It's not hard to do and maintain, but blocked users will still annoy you elsewhere anyway.

**Comment by Justtaway** [comment]
> Or do you just want to hide messages in conversations?

Yeah, just hiding it from conversations in group chats will be enough. Nekogram X have this feature, but it's android only. Still, it's very useful in certain situations.
UPD: actually, Nekogram X removed this feature, saying it violates ToS. Is it true?

**Comment by famewolf** [comment]
I have to wonder what sort of terms of service thinks it's ok for one user to harass another in group chats with no recourse.  Came here specifically hoping to find this features for some peace and quiet.

**Comment by babyrig** [comment]
> UPD: actually, Nekogram X removed this feature, saying it violates ToS. Is it true?

Till today this feature works on Nekogram, the only app who has it to Android that I can find. 

I am looking for a Desktop version with this feature do you have any luck so far?
